src/features/staking/subcomponents/StakableBar.tsx

```diff
@@ -32,53 +26,35 @@ export const StakableBar: FC<StakableBarProps> = ({ mode }) => {
 
   return (
     <div className="flex w-full flex-col space-y-3">
-      {mode === "stake" && (
-        <>
-          <div className="m-auto mt-6 flex h-3 w-full flex-row rounded-full">
-            <div
-              style={{ flexBasis: `${unstakablePercent}%` }}
-              className="checkered-blue rounded-l-full"
-            ></div>
-            <div
-              style={{ flexBasis: `${stakedPercent}%` }}
-              className="bg-accent-blue"
-            ></div>
-            <div
-              style={{ flexBasis: `${stakablePercent}%` }}
-              className="rounded-r-full bg-accent-gray"
-            ></div>
-          </div>
-          <div className="flex flex-row items-center">
-            <div className="checkered-blue rounded-full">
-              <BsCircleFill className="text-transparent" />
-            </div>
-            <span className="mx-2">{unstakable}</span>
-            unstakable
-          </div>
-          <div className="flex flex-row items-center">
-            <BsCircleFill className="text-blue-500" />
-            <span className="mx-2">{staked}</span>staked
-          </div>
-          <div className="flex flex-row items-center">
-            <BsCircleFill className="text-accent-gray" />
-            <span className="mx-2">{stakable}</span>stakable
-          </div>
-        </>
-      )}
-      {mode === "unstake" && (
-        <>
-          <div className="m-auto mt-6 flex h-3 w-[100%] flex-row rounded-full">
-            <div
-              className={twJoin(["w-full rounded-full bg-accent-gray"])}
-              // style={{ flexBasis: `${stakablePercent}%` }}
-            ></div>
-          </div>
-          <div className="flex flex-row items-center">
-            <BsCircleFill className="text-accent-gray" />
-            <span className="mx-2">{stakable}</span>stakable
-          </div>
-        </>
-      )}
+      <div className="m-auto mt-6 flex h-3 w-full flex-row rounded-full">
+        <div
+          style={{ flexBasis: `${unstakablePercent + 3}%` }}
+          className="checkered-blue rounded-l-full"
+        ></div>
+        <div
+          style={{ flexBasis: `${stakedPercent}%` }}
+          className="bg-accent-blue"
+        ></div>
+        <div
+          style={{ flexBasis: `${stakablePercent + 3}%` }}
```

Comment:
Why is the +3 needed?

Reply:
I removed the `+3` and added `min-w-[3%]` to TailwindCSS classes. The reason for the extra 3% is because if `unstakable` or `stakable` numbers are too small, the bar will have sharp edges. 

Without the extra 3%:
<img width="296" alt="Screenshot 2023-08-30 at 12 27 45 PM" src="https://github.com/airswap/airswap-voter-rewards/assets/90465420/cc20f746-ce52-48c6-9208-4d13c4a1554a">

with the extra 3%:
<img width="296" alt="Screenshot 2023-08-30 at 12 28 00 PM" src="https://github.com/airswap/airswap-voter-rewards/assets/90465420/53b1620b-dc44-4c97-a942-2a10e8ead386">